frontend/service/oauth.service.js

```diff
@@ -41,10 +44,32 @@ export class OAuthService {
       const accessToken = response.accessToken
       const username = await this.getUsername(accessToken)
       this.$store.commit('auth/login', { username, accessToken })
+      const oauth = {
+        username,
+        accessToken
+      }
+      const signer = 'password'
+      const token = jwt.sign(oauth, signer)
+      this.$cookies.set('oauth', token, {
+        path: '/',
+        maxAge: 60 * 60
+      })
     }
     return response
   }
 
+  autoLoginByCookie () {
+    const token = this.$cookies.get('oauth')
+    try {
+      const decoded = jwt.verify(token, 'password')
+      const username = decoded.username
+      const accessToken = decoded.accessToken
+      this.$store.commit('auth/login', { username, accessToken })
+    } catch (err) {
+      // error logic here
```

Comment:
What types of errors have you detected here? maybe could be interesting to put a `console.error` ?